### DISCUSSION TITLE
Support CSS Container Queries

### DISCUSSION THREAD

**Discussion by tiana25** [rofrischmann/fela#908] [Ideas]
CSS container queries are currently supported in Google Chrome (105) and soon in Safari 16.
It would be great if Fela also had a support for this type of queries. Right now it seems it doesn’t have any:(

**Comment by robinweser** [comment]
Do you want to help make it happen? I'm happy to guide you through the process!
Isn't too much work to do, but a great addition indeed. Would love to see them in action.

**Reply by tiana25** [reply]
Sure, I would love to help with that!

**Reply by benpender** [reply]
@tiana25 @robinweser I’m interested in this as well and could possibly help out. Has there been any progress?

**Reply by robinweser** [reply]
I will get back to both of you by the end of the day.
I only wrote a quick mail answer to @tiana25 a couple of weeks ago, but haven't really been able to really get back to it.
But I now need it too for my own projects, so higher priority now. Turns out adding it is not as easy as I first thought, so might be sth. that I have to implement primarily.

**Reply by benpender** [reply]
@robinweser do you have any update on support here?
